DOCUMENTATION.md

```diff
@@ -61,6 +61,16 @@ To get an API key, [sign in to GOV.UK Notify](https://www.notifications.service.
 
 If you use a proxy, you must set the proxy configuration in the `web.config` file. Refer to the [Microsoft documentation on proxy configuration](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/proxy-configuration) for more information.
 
```

Comment:
Is this another way to create a new instance of the client? If so, I would add in some content to say that there are two ways to do this, and distinguish between the two. I would also add in the reason why you would choose one way over the other, however if this is unnecessary as the user would know this already, happy to not include this.

For example: "You can also create a new instance of the client using a proxied `HttpClient`. Add this code to your application:" or something similar.